### PR TITLE
Updating Readme to replace autocorrect by fix

### PR DESCRIPTION
**Pull request by ezefranca**
Since autocorrect was replaced by fix, I think it should be updated in the documentation also.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -164,7 +164,7 @@ $ swiftlint help
 Available commands:
 
    analyze         [Experimental] Run analysis rules
-   autocorrect     Automatically correct warnings and errors
+   fix           Automatically correct warnings and errors
```
Comment (jpsim) [inline README.md:167]: This is incorrect. Can you run `swiftlint help` and paste the response here?

File: README.md [hunk 2]
```diff
@@ -175,7 +175,7 @@ Available commands:
 Run `swiftlint` in the directory containing the Swift files to lint. Directories
 will be searched recursively.
 
-To specify a list of files when using `lint`, `autocorrect` or `analyze`
+To specify a list of files when using `lint`, `fix` or `analyze`
```
Comment (jpsim) [inline README.md:178]: This is incorrect.

```suggestion
To specify a list of files when using `lint` or `analyze`
```

**Review by jpsim — CHANGES_REQUESTED** [review]
Thanks for the PR, but there are a few inaccuracies in your changes. Could you please address my feedback?

File: README.md (hunk 1 above)
Comment (ezefranca) [reply README.md:167]: Did it

**Review by ezefranca — COMMENTED** [review]
Did the suggestions

File: README.md (hunk 1 above)
Comment (ezefranca) [reply README.md:167]: I figured out that the `fix` not appear in help, is there any chance to create a issue and/or PR adding?

**Review by ezefranca — COMMENTED** [review]
Did the changes that @jpsim addressed

**Review by jpsim — APPROVED** [review]
I updated the output of `swiftlint help` myself.

**Pull request merged**